ci/cloudbuild/builds/cmake-install.sh

```diff
@@ -27,12 +27,26 @@ export CXX=clang++
 INSTALL_PREFIX="$(mktemp -d)"
 readonly INSTALL_PREFIX
 
+mapfile -t feature_list < <(cat ci/etc/full_feature_list)
+# Add the hand-crafted libraries and custom features to the full list
+feature_list+=(
+  bigtable
+  pubsub
+  spanner
+  storage
+  experimental-storage-grpc
+  pubsublite
```

Comment:
should `pubsublite` be in `ci/etc/full_feature_list` ?

Reply:
Will be, once I add `pubsublite` to the `generator/generator_config.textproto` file.